includes/subscriptions/templates/html-subscriptions-plugin-notice.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Admin WC Subscriptions plugin warning template.
+ *
+ * @package WooCommerce\Payments
+ */
+
+?>
+<script type="text/template" id="tmpl-wcpay-subscriptions-plugin-warning">
+	<div id="wcpay-subscriptions-plugin-warning-notice" class="wc-backbone-modal">
+		<div class="wc-backbone-modal-content">
+			<section class="wc-backbone-modal-main" role="main">
+				<header class="wc-backbone-modal-header">
+					<h1><?php esc_html_e( 'Are you sure?', 'woocommerce-payments' ); ?></h1>
+					<button class="modal-close modal-close-link dashicons dashicons-no-alt">
+						<span class="screen-reader-text">Close modal panel</span>
+					</button>
+				</header>
+				<article>
+					<p>
+						<?php
+							printf(
+								// Translators: placeholders are opening and closing strong HTML tags.
+								esc_html__( 'By deactivating the %1$sWooCommerce Subscriptions%2$s plugin, your store will switch to use %1$sSubscriptions powered by WooCommerce Payments%2$s.', 'woocommerce-payments' ),
```

Comment:
Nitpick: "switch to" or "use" is redundant. We can probably remove one or the other.

...your store will switch to Subscriptions powered by..., or
...your store will use Subscriptions powered by...